### DISCUSSION TITLE
When trying to access a page, I received a "404" error message

### DISCUSSION THREAD

**Discussion by johnfairweather** [community/community#51517] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I just tried to access this page and received a "404" error code, please advise - https://github.com/Jammy2211/autolens_abell_1201

**Comment by CodeMaster7000** [comment, marked as answer]
Hi @johnfairweather,

The reason is that the user has no such repository of that name. Maybe you made a typo?

**Reply by johnfairweather** [reply]
Thank you for your reply,
Accessing this document "https://academic.oup.com/mnras/article/521/3/3298/7085506?login=false", I went down, until I found this link "https://github.com/Jammy2211/autolens_abell_1201".  Clicking on that link, I received the "404" message - Any ideas as to what is wrong?   (I expect the document is in another location, or not yet uploaded.)

**Reply by CodeMaster7000** [reply]
The user might have deleted the repository.

Also, if you found my answer helpful, could you please mark it as an answer? I am trying to get the Galaxy Brain bronze achievement.
